packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -75,6 +77,63 @@ type StyleXStateOptions = $ReadOnly<{
   ...
 }>;
 
+function validateAliases(aliases: any): StyleXOptions['aliases'] {
```

Comment:
Adding any since options has been defined as object of string: mixed, causing issues with the aliases which should be type object but are getting assigned mixed.

Reply:
This should not break any typing since this is just a validation function and returns the currect types for the aliases object.

Reply:
Use `$FlowFixMe` instead of `any`. Other than that, I'll add validation for this. I'm also looking into using `zod` or something similar instead of the hand-rolled solution I have right now.

I'm looking for something with good error messages.

Reply:
Sure @nmn, I've updated the typing for now. I'll also check into zod and try to add validation using that while you are looking into it.